upf/walkers/identitydag.py

```diff
@@ -48,6 +48,12 @@ def walk_implies( self, expression: FNode, args: List[FNode], **kwargs):
     def walk_iff( self, expression: FNode, args: List[FNode], **kwargs):
         return self.manager.Iff(args[0], args[1])
 
+    def walk_exists( self, expression: FNode, args: List[FNode], **kwargs):
+        return self.manager.Exists(args[0], *expression.variables())
+
+    def walk_forall( self, expression: FNode, args: List[FNode], **kwargs):
+        return self.manager.Exists(args[0], *expression.variables())
```

Comment:
```suggestion
        return self.manager.Forall(args[0], *expression.variables())
```

Reply:
Yep, with the next commit walkers should be ok!

Reply:
In the last commit you didn't update the type checker and the pddl writer